oak_attestation_verification/src/verifier.rs

```diff
@@ -34,11 +40,12 @@ const ADDITIONAL_DATA: &[u8] = b"";
 
 /// Verifies entire setup by forwarding to individual setup types.
 pub fn verify(
+    now_utc_millis: i64,
```

Comment:
I don't remember if there was a reason for using an i64 instead of a proper time type? If so, could you add a comment? Otherwise I'm sure I (and others) will keep wondering every time we come across this line :)